components/Buttons/tests/unit/ButtonThemingTest.swift

```diff
@@ -0,0 +1,63 @@
+// Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import XCTest
+
+import MaterialComponents.MaterialButtons
+import MaterialComponents.MaterialColorScheme
+import MaterialComponents.MaterialShapeScheme
+import MaterialComponents.MaterialTypographyScheme
+import MaterialComponentsAlpha.MaterialButtons_Theming
+
+class ButtonThemingTest: XCTestCase {
```

Comment:
_Nit:_ I think our test classes are usually pluralized, i.e. ButtonThemingTests

Reply:
Also consider naming it "ButtonsThemingTests" to align with the component name (Button_s_).